ddclient.in

```diff
@@ -4238,6 +4259,174 @@ sub nic_dnsexit_update {
     }
 }
 ######################################################################
+## nic_dnsexit2_examples
+######################################################################
+sub nic_dnsexit2_examples {
+    return <<"EoEXAMPLE";
+o 'dnsexit2'
+
+The 'dnsexit2' protocol is the new API protocol used by the dynamic hostname services
+of the 'DNSExit' dns services. This is currently used by the free
+dynamic DNS service offered by www.dnsexit.com.
+
+Configuration variables applicable to the 'dnsexit2' protocol are:
+  protocol=dnsexit2         ##
+  password=YourAPIKey       ## API Key of your account.
+  server=api.dnsexit.com    ## defaults to api.dnsexit.com.
+  path=/dns/                ## defaults to /dns/.
+  record-type=A             ## defaults to A record.
```

Comment:
This can presumable only be A or AAAA?

Reply:
Thank you for your review @SuperSandro2000 :+1: .
It technically can be any supported DNS record type by the DNSExit API (TXT, MX, CNAME), but practically for our purposes only A and AAAA make sense (to me at least).